en/docs/administration/customize-the-developer-portal.md

```diff
@@ -0,0 +1,48 @@
+# Customize the Developer Portal
+
+You can share APIs internally and externally through the Choreo Developer Portal. You can change the look and feel of your Developer Portal by changing the theme to match your brand. Doing so will help you give a better developer experience to your users.
+
+To configure the Developer Portal theme, follow the steps given below:
+
+!!! Note
+    - To change the Developer Portal theme for an organization, you need to be an admin user of that organization.<br/><br/>
+    - If you want to remove any theme change you make, you need to reset to the default theme. Choreo does not support undoing a specific change and reverting to the previous theme.
+
+1. Sign in to the Choreo Console at [https://console.choreo.dev/](https://console.choreo.dev/) using a Google, GitHub, or Microsoft account.
+
+2. If you are a new user, create an organization. Enter a unique organization name. For example, "Stark Industries".
+
+    ![Create an organization in Choreo](../assets/img/references/enterprise-login/create-choreo-organization.png){.cInlineImage-small}
+
+3. In the left pane, click **Settings**.
+
+4. In the **Organization** tab, click **Devportal Theme**.
+
+    ![Access Devportal theme](../assets/img/administration/devportal-theme/access-devportal-theme.png)
+
+    Once you access the theme, you can customize the Home page, color theme, font, header and footer, logos, etc., by expanding the **General**, **Home Page**, **Color Palette**, **Typography**, and **Footer Links** sections.
+
+5. Make a change to the theme. For example, let's change the title on the home page and the color theme. 
+
+    1. To update the title on the Home page, expand the **Home Page** section, and in the **Title** field, change the default text (for example, to `Try our APIs!`).
+   
+    2. To update the color theme, expand the **Color Palette** section, and change the colors as required (for example, change the background color to `#C3C5CD` and the primary color of the buttons to `#086634`).
+   
+    3. Click **Preview** to view a preview of the Developer Portal with the changes you made. For example, let's assume you changed the primary color of the background to `#C3C5CD` and the primary color of the buttons to `#086634`. Based on these changes, the preview appears as follows.
+
+         ![Preview of customization](../assets/img/administration/devportal-theme/preview-of-customization.png)
+   
+    4. Click **Save**.
+
+6. Sign in to Choreo Developer Portal at [https://devportal.choreo.dev](https://devportal.choreo.dev).
+
+The Home page will appear as it did in the preview.
+
+## Reset the Developer Portal theme
+
+If you changed the Developer Portal these and want to reset it, follow the steps given below:
+
+1. Access Choreo Console and in the left pane, click **Settings**.
```

Comment:
Let's break it up and  say, 
1. sign in to the...
2. In the left pane..

Reply:
The user has already signed in to the  Console and only needs to move to another tab.

Reply:
In that case, why do we tell them to access the Choreo console?

Reply:
This is for them to switch to the tab where the Choreo Console is running. They don't need to sign in again.

Reply:
Since this is not a tutorial, do we expect they will do all of these in one go? Or refer to these sections as individual sections when the need arises? My opinion is that it is the latter.

Reply:
Agreed. Changed as suggested.